### DISCUSSION TITLE
Global Projects or direct link customers -> activities

### DISCUSSION THREAD

**Discussion by Ruff-G** [kimai/kimai#3501] [Questions & Answers] [answered]
Hello,
is it possible to track times without projects, but with activities (customer -> activity) or would it be possible to create global projects (customer -> project [global] -> activity [global])?

That would be really great for repeated projects and tasks.

**Comment by kevinpapst** [comment, marked as answer]
No. 
Please use the search, this was discussed before. 
In short: mainly technical reasons.
